cmp/grid/GridAutosizeMode.js

```diff
@@ -24,13 +24,13 @@ export const GridAutosizeMode = Object.freeze({
      * In addition to the affordances provided by ON_DEMAND, Grid will autosize columns when
      * the GridModel's sizingMode changes.
      */
-    ON_SIZING_MODE_CHANGE: 'onSizingModeChange'
+    ON_SIZING_MODE_CHANGE: 'onSizingModeChange',
 
-    // COMING SOON
-    // /**
-    //  *  Grid will autosize columns when data is *first* loaded into a grid.   Persisted grids
-    //  * will only resize again if GridModel's sizingMode subsequently changes.
-    //  */
-    // ON_FIRST_LOAD: 'onFirstLoad'
+    /**
```

Comment:
Need to mention more clearly that this is default, if it is, and it can come first in the file.  Also I think the point about "the affordances provided by ON_DEMAND" is secondary and should go last or be removed entirely from the comment.